src/components/OrgPostCard/OrgPostCard.tsx

```
@@ -461,17 +997,47 @@
-                          src={attachment.url}
+                          src={
+                            sanitizeUrl(
+                              filePreviewUrls[attachment.objectName] ||
+                                previewUrls[attachment.objectName] ||
+                                '',
+                            ) || ''
+                          }
                           type={attachment.mimeType}
                         />
-                        {t('videoNotSupported')}
                       </video>
```

Biome (1.9.4)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)